docs/reference/api/docker_remote_api_v1.20.md

```diff
@@ -208,6 +209,7 @@ Json Parameters:
 -   **CpusetCpus** - String value containing the `cgroups CpusetCpus` to use.
 -   **CpusetMems** - Memory nodes (MEMs) in which to allow execution (0-3, 0,1). Only effective on NUMA systems.
 -   **BlkioWeight** - Block IO weight (relative weight) accepts a weight value between 10 and 1000.
+-   **MemorySwappiness** - Integer value, tuning container's memory swappiness. Accepts a value between 0 and 100.
```

Comment:
Tune a container's memory swappiness behavior. Accepts an integer between 0 to 100.